init.lua

```diff
@@ -367,7 +367,7 @@ function Server.new(lang, root, cmd, init_options)
 				--	honorsChangeAnnotations = true
 				-- },
 				publishDiagnostics = {
-					--	relatedInformation = true,
+					relatedInformation = true,
```

Comment:
Does this key just need to exist? Or is a `true` value necessary? Since this module does not process the `relatedInformation` in `Diagnostic`s (https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#diagnostic), this value should be `false` if the key needs to exist.

Reply:
I just tested it, yes it just needs to exist. `false` works without problems too. Without the key zettelkasten/markdown language server `zk` simply fails to start.

Reply:
Okay, I've marked it as `false` in https://github.com/orbitalquark/textadept-lsp/commit/234f8297f13e30049a36f0d0f1acf6b31f6f2a00.